src/feature/featurelistcallback.h

```diff
@@ -37,7 +37,7 @@ struct FeatureSupportedPlatforms {
   bool wasm = false;
 };
 
-consteval auto byPlatform(FeatureSupportedPlatforms support) {
```

Comment:
This seems like a downgrade :D Why that?

Reply:
As a workaround for [QTBUG-132735](https://bugreports.qt.io/browse/QTBUG-132735) we downgraded to C++17 when building `.deb` packages, and `consteval` is only available in C++20 and beyond.